src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,33 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (storageDao.get(user.getLogin()) != null) {
+            throw new RuntimeException("User is existed");
+        }
+        if (user.getLogin() == null) {
+            throw new RuntimeException("Login can't be null");
+        }
+        if (user.getLogin() == "") {
```

Comment:
You can also use method "isBlank" here
```suggestion
        if (user.getLogin().isBlank()) {
```